_themes/conan_theme/layout.html

```diff
@@ -204,8 +204,13 @@
         <div class="rst-content">
         {%- endif %}
           {% if current_version.startswith("1") %}
-          <div class="admonition warning wy-text-center" style="position:sticky;top:0;border: 2px solid red;background-color: #ffcaca;">
-            <p>This document is for a <b>"1.X" legacy</b> Conan version, <b>which is no longer recommended</b>. Please update to Conan 2, <a href="https://docs.conan.io/2/">click here to read the <b>Conan 2</b> documentation</a></p>
+          <div class="admonition warning wy-text-center" style="position:sticky;top:0;background: #fcfcfc;padding: inherit" id="conan-banners">
+            <div style="border: 2px solid red;background-color: #ffcaca;padding: 3px;margin-bottom: 5px">
+              <p>This document is for a <b>"1.X" legacy</b> Conan version, <b>which is no longer recommended</b>. Please update to Conan 2, <a href="https://docs.conan.io/2/">click here to read the <b>Conan 2</b> documentation</a></p>
+            </div>
+            <div style="border: 2px solid red;background-color: #ffff00;padding: 3px">
```

Comment:
Sorry, I didn't realize. If we put this outside of ``{% if current_version.startswith("1") %}`` it will also display in Conan 2 docs? Do we want to display it too in Conan 2 docs? I'd say yes, users reading Conan 2 docs might be migrating from Conan 1.